onboarding/mentors.md

```diff
@@ -45,9 +43,15 @@ Get to know each other. This can be a nice, casual conversation. Focus on active
 
 ### Your first week
 - Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
+  - Be available to troubleshoot checklist items.
+  - Encourage (pair) on documenting problems with the checklist.
 - Look for chances to pair or work alongside one another throughout the week.
   - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
   - If they haven't set up hokusai yet, consider pairing on that.
+  - Notice what types of tasks they are good at and connect them to engineers who share that strength. (https://dev.to/flippedcoding/how-to-be-a-good-software-engineer-mentor-4b)
+  - Notice what types of tasks they struggle with and connect them to engineers who can help. (https://dev.to/flippedcoding/how-to-be-a-good-software-engineer-mentor-4b)
+  - Encourage improvements to READMEs while setting up projects.
+- Take your onboardee to team meetings and explain any business context after attending.
```

Comment:
This is a good place to be explicit about expectations if you're remote!

i.e. "Take your onboardee to team meetings and explain any business context after attending. If you're in the office, drop by their desk and bring them in-person. If you're remote, make a point to slack them to remind." (or something like that)